exporter/fileexporter/file_exporter.go

```diff
@@ -72,10 +73,21 @@ func exportMessageAsLine(e *fileExporter, message proto.Message) error {
 func (e *fileExporter) Start(context.Context, component.Host) error {
 	var err error
 	e.file, err = os.OpenFile(e.path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
+	if err == nil {
```

Comment:
Technically you don't need to set this as the default value is `false`, although this can probably be kept as an explicit statement. Not very important though ;)

Reply:
Thanks. I actually would like to remove the flag if the ShareComponent can guarantee `Shutdown` is called only once. Otherwise, I think an explicit statement would be better :)